_posts/2016-01-24-load-averages.md

```diff
@@ -16,37 +16,37 @@ it has to do.
 When speculating about high load averages it's important to remember **load averages are the average number of *ready
 to run / running* processes in the run queue.**
```

Comment:
Maybe say something about the Run Queue/Load average being shared across the number of CPU's? e.g. A load average of 4 on an 8 core machine compared to a load average of 4 on a 2 core machine.